livy/src/test/java/org/apache/zeppelin/livy/LivyInterpreterIT.java

```diff
@@ -157,7 +157,7 @@ public void testSparkInterpreterRDD() {
     }
   }
 
-  @Test
+//  @Test
```

Comment:
Need to disable it as it could cause OOM in CI due to docker container memory limitation